code/game/objects/structures/orbital_cannon.dm

```diff
@@ -161,6 +161,14 @@
 
 	update_icon()
 
+/// Handles the playing of the Orbital Bombardment incoming sound, usually a spiraling whistle noise but can be overridden.
+/obj/structure/orbital_cannon/proc/ob_pre_impact_sound(target, var/ob_sound = 'sound/effects/OB_incoming.ogg')
```

Comment:
This is a bad proc name, it does more than just play sounds on the ground, it also has an animation. It should have a more appropriate name.